services/movie.py

```diff
@@ -0,0 +1,38 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> QuerySet:
+    query_set_movies = Movie.objects.all()
+
+    if genres_ids and actors_ids:
+        query_set_movies = query_set_movies.filter(genres__id__in=genres_ids,
+                                                   actors__id__in=actors_ids)
```

Comment:
The filtering logic here will only return movies that match both the genre and actor IDs. If you want to filter by either condition, you should use `Q` objects to combine filters with an OR condition. Otherwise, this condition will override the subsequent individual filters.